ansible/era1.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Verify Ansible versions
+  hosts: all
+  tags: always
+  become: false
+  run_once: true
+  gather_facts: false
+  tasks:
+    - local_action: command ./roles.py --check
+      changed_when: false
+
+- name: Configure ERA1 files hosting
```

Comment:
You don't need this file. It's exactly the same as `ansible/era.yml` file.

Remember that you're configuring `nginx_sites` for the whole host for one run of `infra-role-nginx`, you don't need multiple playbooks to run the same thing.

Reply:
Yes.